lib/querystring.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+const {
+  StorageObject,
+  percentEscape,
+  unescapeBuffer
+} = require('internal/querystring');
 const QueryString = module.exports = {
   unescapeBuffer,
```

Comment:
I'd almost rather just document this IMHO. I've personally had uses for some of these in my own projects. \*shrug\*